meta-balena-common/recipes-connectivity/openssh/openssh_%.bbappend

```diff
@@ -14,8 +14,8 @@ FILES_${PN}-sshd += " \
 "
 
 do_install_append () {
-    # Advertise SSH service using an avahi service file                                                                                                                                                                                                                                                                                                                   
-    mkdir -p ${D}/etc/avahi/services/                                                                                                                                                                                                                                                                                                                                                    
```

Comment:
Nice.

Reply:
forgot to mention in the commit message. some whitespace fixes too..

Reply:
o. did i miss the diff. lol

Reply:
Don't worry :)